app/components/transaction.js

```diff
@@ -181,6 +181,7 @@ function transactionMobile(transaction) {
         h('span', valueStr),
         h('span', ' wei'),
       ]),
+      h('p', `Nonce: ${txParams.nonce} ` + (txParams.nonce ? `(${parseInt(txParams.nonce)})` : '')),
```

Comment:
Do we need to set the `radix` to 16 since it's a hex value we're converting?

Reply:
hi @matthewwalsh0 thank you for your review :pray:  About this line, I've followed the pattern in the Extension transactions above (copied the same): https://github.com/MetaMask/state-log-explorer/blob/main/app/components/transaction.js#L102

So in case we need to change it, I think it's good to change them both. I've seen that: `Numbers that start with 0x use a radix of 16 (hexidecimal)` so maybe it is not strictly needed? Let me know what you think is best

https://stackoverflow.com/questions/6611824/why-do-we-need-to-use-radix-parameter-when-calling-parseint

Reply:
Thanks Mariona, didn't realise it defaulted based on the `0x`!

If it's done the same elsewhere here, consistency makes sense.

Reply:
thank you!! It's been good to double check this :pray: